chain/bitcoin/utxo.go

```diff
@@ -183,7 +183,7 @@ func (tx *Tx) Sign(signatures []pack.Bytes65, pubKey pack.Bytes) error {
 				continue
 			}
 		} else {
-			if txscript.IsPayToWitnessScriptHash(sigScript) || txscript.IsPayToWitnessScriptHash(sigScript) {
+			if txscript.IsPayToWitnessPubKeyHash(sigScript) || txscript.IsPayToWitnessScriptHash(sigScript) {
```

Comment:
I probably don't have the context but was our old bitcoin implementation broken?

Reply:
It didn't affect our internal usage, but may have affected external users since that particular branch wasn't being handled.